backends/vulkan/targets.bzl

```diff
@@ -5,7 +5,14 @@ load("@fbsource//tools/build_defs:platform_defs.bzl", "ANDROID", "CXX", "FBCODE"
 
 
 def get_vulkan_compiler_flags():
-    return ["-Wno-missing-prototypes", "-Wno-global-constructors"]
+    return select({
+        "DEFAULT": [
+            "-Wno-global-constructors",
+            "-Wno-missing-prototypes",
+            "-Wno-sign-compare",
+        ],
+        "ovr_config//os:windows": [],
```

Comment:
I _think_ we can leave the existing `#pragma` stuff and just add a `select` call with a windows option like you did here so that unsupported options are not used when compiling with Windows

Reply:
for sure, I can do that!